src/services/sync-checker.ts

```diff
@@ -8,6 +8,8 @@ const logger = require('../services/logger')
 
 import axios from 'axios'
 
+const DEFAULT_SYNC_ALLOWANCE: number = parseInt(process.env.DEFAULT_SYNC_ALLOWANCE) || 5
```

Comment:
I'd rather have all of the .env variables checked in one place and rather than silently fallback to a default, I'd rather it error out and tell you that you made a dumb mistake and forgot the var.

Probably around here: https://github.com/pokt-foundation/portal-api/blob/bddc518b3d833a088e787f1823587313b7ba33e1/src/application.ts#L59

Reply:
I like having `.env` vars in one place—[this](https://github.com/pokt-foundation/portal/blob/mvp/packages/backend/src/environment.ts) is how I usually do it

Reply:
That does look nicer.

Reply:
yep, like it too

Reply:
for now I'm doing a quick fix but an issue can be opened for that to change all the envs